src/server/nuclearnet/fake_nuclearnet_client.ts

```diff
@@ -29,8 +36,9 @@ export class FakeNUClearNetClient implements NUClearNetClient {
   public connect(options: NUClearNetOptions): () => void {
     this.peer = {
       name: options.name,
-      address: '127.0.0.1',
-      port: 7447,
+      // Address and port are not used for anything yet, dummy values used.
+      address: 'fake_address',
+      port: 0,
```

Comment:
Exactly! 😛  What if -1 gets cast to a Uint16 and wraps to 65535! 😆 

I think I'm actually going to use `NaN` lol